src/status_im/subs/general.cljs

```diff
@@ -120,12 +120,12 @@
 (re-frame/reg-sub
  :dimensions/window-width
  :<- [:dimensions/window]
- :width)
+ :-> :width)
 
 (re-frame/reg-sub
  :dimensions/window-height
  :<- [:dimensions/window]
- :height)
+ :-> :height)
```

Comment:
Changed this style, based on re-frame reg-sub docs:


... 
For example, a significant number of subscriptions exist only to get a value
from the `input-values`. As shown below, this subscription will simply retrieve
the value associated with the `:foo` key in our db:

```clojure
(reg-sub
 :query-id
 (fn [db _]    ;; :<---- trivial boilerplate we might want to skip over
   (:foo db)))
```

This is slightly more boilerplate than we might like to do,
as we can use a keyword directly as a function, and we might like to do this:

```clojure
(reg-sub
 :query-id
 :foo)  ;; :<---- This could be dangerous. If `:foo` is not in db, we get the `query-vector` instead of `nil`.
```

By using `:->` our function would not contain the `query-vector`, and any
missing keys would be represented as such:

```clojure
(reg-sub
 :query-id
 :-> :foo)
```

This form allows us to ignore the `query-vector` if our `computation-function`
        has no need for it, and be safe from any accidents. Any 1-arity function can be provided,
and for more complicated use cases, `partial`, `comp`, and anonymous functions can still be used.

cc: @flexsurfer 
I've seen this is mostly used in the legacy code, should we remove them in a later refactor?
I believe `(rn/get-window)` is able to give us the same results without need to use re-frame.